includes/Dashboard.php

```diff
@@ -318,7 +317,7 @@ public function get_dashboard_settings() {
 				'assetsURL'          => trailingslashit( WEBSTORIES_ASSETS_URL ),
 				'cdnURL'             => trailingslashit( WEBSTORIES_CDN_URL ),
 				'version'            => WEBSTORIES_VERSION,
-				'encodeMarkup'       => $this->supports_decoding(),
+				'encodeMarkup'       => ( new \Google\Web_Stories\Decoder( $this->experiments ) )->supports_decoding(),
```

Comment:
I think we should create the instance as a class property in the constructor.

Also, I dont think we need namespace here.